src/main.cu

```diff
@@ -29,7 +29,7 @@
 #define VIS_MODE 1
 
 // Number of steps and repetitions for different expeirments.
-#define VIS_REPETITIONS 1
+#define VIS_REPETITIONS 3
```

Comment:
Why are repetitions required when visusalisting a single simulation?

Reply:
As there is a non visualisation experiment which runs with vis off that generates non performance related data to demonstrate the models behaviour.

Reply:
Then why are the `#defines` prefixed with `VIS` here?

Reply:
Ok given this a better prefix.